src/mscorlib/shared/Internal/Runtime/CompilerServices/Unsafe.cs

```diff
@@ -168,6 +168,46 @@ public static bool AreSame<T>(ref T left, ref T right)
             // ret
         }
 
+        /// <summary>
+        /// Determines whether the memory address referenced by <paramref name="left"/> is greater than
+        /// the memory address referenced by <paramref name="right"/>.
+        /// </summary>
+        /// <remarks>
+        /// This check is conceptually similar to "(void*)(&amp;left) &gt; (void*)(&amp;right)".
+        /// </remarks>
+        [Intrinsic]
+        [NonVersionable]
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Is Aggressive inlining necessary or just there for good measure?

Reply:
It's the same pattern used by the other methods on the type - I assume for good measure?